TradeItIosTicketSDK2/TradeItYahooTradingTicketViewController.swift

```diff
@@ -20,11 +21,16 @@ class TradeItYahooTradingTicketViewController: CloseableViewController, UITableV
             assertionFailure("ERROR: Could not instantiate TradeItSelectionViewController from storyboard")
             return
         }
+        guard let accountSelectionViewController = self.viewProvider.provideViewController(forStoryboardId: .yahooAccountSelectionView) as? TradeItYahooAccountSelectionViewController else {
+            assertionFailure("ERROR: Could not instantiate TradeItYahooAccountSelectionViewController from storyboard")
```

Comment:
`assertionFailure` will not trigger in a production build. Is this what you intended? Or did you mean to use `preconditionFailure`?